priv/repo/migrations/20230608105745_add_deregistered_at_field_db_rm.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Trento.Repo.Migrations.AddDeregisteredAtFieldDbRm do
```

Comment:
Change filename to: `xxx_add_deregistered_at_to_database_read_model` and class name `AddDeregisteredAtToDatabaseReadModel`